tests/integration_tests/test_game.py

```diff
@@ -391,6 +392,55 @@ async def test_game_with_foed_player(lobby_server):
         await join_game(guest_proto, game_id)
 
 
+@fast_forward(60)
+async def test_game_join_log(lobby_server, database):
+    _, _, host_proto = await connect_and_sign_in(
+        ("test", "test_password"), lobby_server
+    )
+    guest_id, _, guest_proto = await connect_and_sign_in(
+        ("Rhiza", "puff_the_magic_dragon"), lobby_server
+    )
+    await read_until_command(guest_proto, "game_info")
+    await read_until_command(host_proto, "game_info")
+
+    # Host game
+    await host_proto.send_message({
+        "command": "game_host",
+        "mod": "faf",
+        "visibility": "public",
+    })
+    msg = await read_until_command(host_proto, "game_launch")
+    game_id = int(msg["uid"])
+
+    msg = await read_until_command(host_proto, "game_info")
+
+    assert msg["hosted_at"] is None
+    assert msg["launched_at"] is None
+
+    await open_fa(host_proto)
+    await read_until_command(host_proto, "HostGame", target="game")
+
+    msg = await read_until_command(host_proto, "game_info")
+    hosted_at = msg["hosted_at"]
+    assert datetime.fromisoformat(hosted_at) <= datetime_now()
+    assert msg["launched_at"] is None
```

Comment:
Since this test is about testing the game join log, not the game hosting messages, I think it would be better to use the `host_game` helper here as it will simplify things a bit.
```suggestion
    game_id = await host_game(host_proto)
```

if some of these intermediate assertions are missing from the game host tests we should add them there.